tests/environment-hook.bats

```diff
@@ -29,6 +29,26 @@ load '/usr/local/lib/bats/load.bash'
   unstub vault
 }
 
+@test "Load default env file containing secrets with special characters from vault server" {
+  export BUILDKITE_PLUGIN_VAULT_SECRETS_SERVER=https://vault_svr_url
+  export BUILDKITE_PLUGIN_VAULT_SECRETS_DUMP_ENV=true
+  export TESTDATA="MY_SECRET=\"|- :fooblah\""
```

Comment:
the actual test should have a space after the colon:
```suggestion
  export TESTDATA="MY_SECRET=\"|- : fooblah\""
```

(remember to update the expected value of the test as well)